socorro/tests/external/pubsub/test_crashqueue.py

```diff
@@ -109,6 +95,6 @@ def test_publish_many(self, pubsub_helper, queue):
         crashqueue.publish(queue, [crash_id_3])
 
         published_crash_ids = pubsub_helper.get_published_crashids(queue)
-        assert sorted(published_crash_ids) == sorted(
-            [crash_id_1, crash_id_2, crash_id_3]
-        )
+        assert set(published_crash_ids) == {
+            crash_id_1, crash_id_2, crash_id_3
+        }
```

Comment:
The original version of this test is curious. I would think we'd want to assert the order of the crash ids published is correct, but I guess this is fine.